x/stakeibc/client/cli/tx_test.go

```diff
@@ -0,0 +1,308 @@
+		f, err := os.CreateTemp("", "")
+		require.NoError(t, err)
+		defer f.Close()
+		_, err = f.WriteString("This is not JSON")
+		require.NoError(t, err)
+
+		args := []string{
+			"[host-zone]",
+			f.Name(),
+		}
+
+		cmd := cli.CmdAddValidators()
+		ExecuteCLIExpectError(t, cmd, args, `invalid character 'T' looking for beginning of value`)
+	})
+	t.Run("wrong json format", func(t *testing.T) {
+		f, err := os.CreateTemp("", "")
+		require.NoError(t, err)
+		defer f.Close()
+		_, err = f.WriteString(`{"blabla_validator_weights":[{"address":"cosmosXXX","weight":1}]}`)
+		require.NoError(t, err)
+
+		args := []string{
+			"[host-zone]",
+			f.Name(),
+		}
+
+		cmd := cli.CmdAddValidators()
+		ExecuteCLIExpectError(t, cmd, args, `invalid creator address (empty address string is not allowed): invalid address`)
+	})
+}
+
+func TestCmdChangeValidatorWeight(t *testing.T) {
+	args := []string{
+		"[host-zone]",
+		"[address]",
+		"[weight]",
+	}
+
+	cmd := cli.CmdChangeValidatorWeight()
+	ExecuteCLIExpectError(t, cmd, args, `unable to cast "[weight]" of type string to uint64`)
+}
+
+func TestCmdChangeMultipleValidatorWeight(t *testing.T) {
+	t.Run("no file", func(t *testing.T) {
+		args := []string{
+			"[host-zone]",
+			"[validator-list-file]",
+		}
+
+		cmd := cli.CmdChangeMultipleValidatorWeight()
+		ExecuteCLIExpectError(t, cmd, args, `open [validator-list-file]: no such file or directory`)
+	})
+	t.Run("empty file", func(t *testing.T) {
+		f, err := os.CreateTemp("", "")
+		require.NoError(t, err)
+		defer f.Close()
+
+		args := []string{
+			"[host-zone]",
+			f.Name(),
+		}
+
+		cmd := cli.CmdChangeMultipleValidatorWeight()
+		ExecuteCLIExpectError(t, cmd, args, `unexpected end of JSON input`)
+	})
+	t.Run("non json file", func(t *testing.T) {
+		f, err := os.CreateTemp("", "")
+		require.NoError(t, err)
+		defer f.Close()
+		_, err = f.WriteString("This is not JSON")
+		require.NoError(t, err)
+
+		args := []string{
+			"[host-zone]",
+			f.Name(),
+		}
+
+		cmd := cli.CmdChangeMultipleValidatorWeight()
+		ExecuteCLIExpectError(t, cmd, args, `invalid character 'T' looking for beginning of value`)
+	})
+	t.Run("wrong json format", func(t *testing.T) {
+		f, err := os.CreateTemp("", "")
+		require.NoError(t, err)
+		defer f.Close()
+		_, err = f.WriteString(`{"blabla_validator_weights":[{"address":"cosmosXXX","weight":1}]}`)
+		require.NoError(t, err)
+
+		args := []string{
+			"[host-zone]",
+			f.Name(),
+		}
+
+		cmd := cli.CmdChangeMultipleValidatorWeight()
+		ExecuteCLIExpectError(t, cmd, args, `invalid creator address (empty address string is not allowed): invalid address`)
+	})
+}
+
+func TestCmdDeleteValidator(t *testing.T) {
+	// no input parsing to test
+}
+
+func TestCmdRestoreInterchainAccount(t *testing.T) {
+	// no input parsing to test
+}
+
+func TestCmdUpdateValidatorSharesExchRate(t *testing.T) {
+	// no input parsing to test
+}
+
+func TestCmdCalibrateDelegation(t *testing.T) {
+	// no input parsing to test
+}
```

Comment:
same thing here, I feel like we may as well just remove these

Reply:
Done 74971a040